src/openms/include/OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/CoarseIsotopePatternGenerator.h

```diff
@@ -87,8 +89,14 @@ namespace OpenMS
     */
     void setMaxIsotope(const Size& max_isotope);
 
+    /// sets the calc_mass_ flag to calculate and return expected masses (true) or atomic numbers (false).
+    void setCalcMass(const bool calc_mass);
```

Comment:
I am unhappy about the term "expected masses", maybe use "accurate monoisotopic mass"

Reply:
maybe even explain the two options in the class description and explain the default behavior (true)